edge-modules/iotedge-diagnostics-dotnet/docker/linux/amd64/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG base_tag=3.1.4-alpine3.11
+FROM mcr.microsoft.com/dotnet/core/runtime:${base_tag}
+
+ARG EXE_DIR=.
+
+ENV MODULE_NAME "DiagnosticsModule.dll"
```

Comment:
why "DiagnosticsModule.dll" is different from project name IotedgeDiagnosticsDotnet?

Reply:
I guess it should be IotedgeDiagnosticsDotnet, I didn't notice since its never run normally. I should probably change it to an echo that says as much